### DISCUSSION TITLE
brew install openjdk fails

### DISCUSSION THREAD

**Discussion by landonh12** [Homebrew/discussions#3267] [Everyday usage] [answered]
Can't install openjdk.

landon@Landon-MBP downloads % brew install openjdk
==> Downloading https://ghcr.io/v2/homebrew/core/openjdk/manifests/18.0.1
Already downloaded: /Users/landon/Library/Caches/Homebrew/downloads/b588207d6269383f0a18dbf2c773ad7d0e046f7b5d17e639ec28bf20e9754353--openjdk-18.0.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/openjdk/blobs/sha256:acb7d100bd61aa51fd0014339897dc8b6405f5c62adf8551cbffe28c1ce6aac7
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:acb7d100bd61aa51fd0014339897dc8b6405f5c62adf8551cbffe28c1ce6aac7?se=2022-05-13T18%3A15%3A00Z&sig=fdV
######################################################################## 100.0%Error: openjdk: Failed to download resource "openjdk"
Download failed: https://ghcr.io/v2/homebrew/core/openjdk/blobs/sha256:acb7d100bd61aa51fd0014339897dc8b6405f5c62adf8551cbffe28c1ce6aac7

**Comment by SMillerDev** [comment]
What does `brew doctor` and `brew config` say?

**Reply by landonh12** [reply]
landon@Landon-MBP ~ % brew doctor
Your system is ready to brew.
landon@Landon-MBP ~ % brew config
HOMEBREW_VERSION: 3.4.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: d0a0bbef8db45cca303c607037338d02d8eaa8ca
Last commit: 2 weeks ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 8c48b914d20901affef2680b048ff19bd4893f39
Core tap last commit: 12 days ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 16-core 64-bit kabylake
Clang: 13.1.6 build 1316
Git: 2.32.0 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.3.1-x86_64
CLT: 13.3.1.0.1.1648687083
Xcode: N/A

**Comment by landonh12** [comment, marked as answer]
Nevermind, the command `ulimit -S 2048` in my .zshenv was causing this issue.
